src/components/QuickGrid/QuickGrid.Props.ts

```diff
@@ -4,6 +4,7 @@ import { ITooltipProps } from '../Tooltip/Tooltip.props';
 export interface IQuickGrid {
     scrollToRow(index: number): void;
     updateColumnWidth(columnIndex: number, getWidth: (oldWidth: number) => number): void;
+    formatBoolCell(column: GridColumn, cellData: any): any;
```

Comment:
is this to get a custom text value?

Reply:
if so, we could move this in the gridcolumn and use it for all data types, not just boolean.

Reply:
Changed the function so it can be referenced via the existing cellFormatter. Although duplicating the code was now neccessary